### PR TITLE
Text classification

### PR DESCRIPTION
**Pull request by KseniiaSych**
8 WIP

### CODE REVIEW

File: 7_text_classification/Text_binary_classification.md [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+    
+    training_args = TrainingArguments(
+        output_dir=output_dir,
+        eval_strategy = "steps",
+        eval_steps = 500,
+        save_strategy = "steps",
+        save_steps=500,
+        learning_rate=2e-5,
+        per_device_train_batch_size=batch_size,
+        per_device_eval_batch_size=batch_size,
```
Comment (milkisbad) [inline 7_text_classification/Text_binary_classification.md:344]: eval batch size can be much greater since you do not backtrack the gradient. can make it like `10*batch_size`

File: 7_text_classification/Text_binary_classification_with_adapters.md [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+    adapters.init(model)
+    config = LoRAConfig(selfattn_lora=True, intermediate_lora=True, output_lora=True, attn_matrices=["q", "k", "v"], alpha=16, r=64, dropout=0.1)
+    model.add_adapter(adapter_name_train_name, config=config)
+    model.train_adapter(adapter_name_train_name)
+    return model
+
+def compute_metrics(eval_pred):
+    predictions, labels = eval_pred
+    predictions = np.argmax(predictions, axis=1)
+    accuracy = evaluate.load("accuracy")
```
Comment (milkisbad) [inline 7_text_classification/Text_binary_classification_with_adapters.md:153]: shouldn't be a problem in this case, but it's good to have a metric independent of a threshold for situations where the model is good, but the threshold is not.

I like to use: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.average_precision_score.html

**Review by milkisbad — COMMENTED** [review]
reveiw changes
